feature-libs/product-configurator/rulebased/occ/variant/variant-configurator-occ.models.ts

```diff
@@ -19,12 +19,25 @@ export namespace OccConfigurator {
   }
 
   export interface Prices {
-    configId?: string;
+    configId: string;
+    attributes?: Supplements[];
     pricingError?: boolean;
     showDeltaPrices?: boolean;
     priceSummary?: PriceSummary;
   }
 
+  export interface Supplements {
```

Comment:
I would suggest to call the interface AttributeSupplements to make it consistent with ValueSupplements below.